__tests__/integration.test.js

```diff
@@ -0,0 +1,35 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+describe("GET", () => {
+  describe("/api/topics", () => {
+    test("responds with 200 status code", () => {
+      return request(app).get("/api/topics").expect(200);
+    });
+    test("response contains array of obj with expected keys", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+          expect(response.body.topics).toHaveLength(3);
+          response.body.topics.forEach((topic) => {
+            expect(topic).toMatchObject({
+              description: expect.any(String),
+              slug: expect.any(String),
+            });
+          });
+        });
+    });
+  });
+});
```

Comment:
You could add a test and functionality for what happens when a bad path is used like `/api/notAnEndpoint`